config/resnet_train.yaml

```diff
@@ -12,7 +12,7 @@ defaults:
     - tasks/train: train_resnet
     - optimizers@tasks.train.optimizers: adam
     - sampler@tasks.train.data_loaders.train.sampler: subset_random
-    - sampler@tasks.train.data_loaders.validation.sampler: subset_sequential
+    - sampler@tasks.train.data_loaders.validation.sampler: subset_random
```

Comment:
Can you remind me the reason for using random sampler in validation, instead of sequential?

Reply:
If I recall correctly, previously the indices were stored in a shuffled form. The indices are now stored consecutively, so unless we shuffle, the dataloader will run through batches consisting only of electrons, then only gammas etc.

Reply:
Oh yeah, that's right